packages/search-input/src/SearchResultGroup/SearchResultGroup.tsx

```diff
@@ -25,10 +25,12 @@ export const SearchResultGroup = ({
 
   return (
     <div>
+      {/* @ts-expect-error li/div incongruency */}
```

Comment:
We should fix this

Reply:
I think this is an issue with Polymorphic 
```
Type '{ children: Element; slot?: string | undefined; style?: CSSProperties | undefined; title?: string | undefined; "aria-label"?: string | undefined; "aria-labelledby": string; ... 263 more ...; as: "li"; }' is not assignable to type 'IntrinsicAttributes & InferredPolymorphicPropsWithRef<"li", InputOptionProps>'.
  Type '{ children: Element; slot?: string | undefined; style?: CSSProperties | undefined; title?: string | undefined; "aria-label"?: string | undefined; "aria-labelledby": string; ... 263 more ...; as: "li"; }' is not assignable to type 'InheritedProps<"li", InputOptionProps>'.
    Types of property 'onClick' are incompatible.
      Type 'MouseEventHandler<HTMLDivElement> | undefined' is not assignable to type 'MouseEventHandler<HTMLLIElement> | undefined'.
        Type 'MouseEventHandler<HTMLDivElement>' is not assignable to type 'MouseEventHandler<HTMLLIElement>'.
          Type 'HTMLDivElement' is missing the following properties from type 'HTMLLIElement': type, valuets(2322)
          ```